inc/saiport.h

```diff
@@ -2104,6 +2104,44 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_SUPPORTED_LINK_TRAINING_MODE,
 
+    /**
+     * @brief Enable ARS on port. This enables the control for quality measure in switch pipeline.
+     *
+     * @type bool
+     * @flags CREATE_AND_SET
+     * @default false
+     */
+    SAI_PORT_ATTR_ARS_ENABLE,
+
+    /**
```

Comment:
How does enabling the ARS on a port relate to ACL action or lookup-based action (ECMP with ASR object) ?

Reply:
I have added ARS object ID as an ACL action besides an explicit action of disabling ARS when being processed via nexthopgroup.
Now operator can enable/disable ARS with an ACL rule for a granular control per flow or a port.